Dockerfile

```diff
@@ -83,9 +83,10 @@ EXPOSE 3000
 ENV NODE_ENV=docker
 
 # Prepare WATT (download node modules, emsdk, build DE ).
-RUN ./launch --dry && \
-     npm cache clean --force && \
-     rm -rf tools/wabt/out/
+# This affects docker image size, can be used to speed up container start.
+# RUN ./launch --dry
+#     npm cache clean --force && \
```

Comment:
shouldn't we clean npm cache on docker image start? Maybe this is the reason why image starts so fast?

Reply:
npm cache clean is optional step and can be considered while docker image creation only.
npm cache clean is not needed once contatner starts since it does not affect image size